test-utils/runSkuScriptInDir.js

```diff
@@ -1,28 +1,35 @@
+const path = require('path');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
 const gracefulSpawn = require('../packages/sku/lib/gracefulSpawn');
-const { exec } = require('child-process-promise');
 
-const skuBin = `${__dirname}/../packages/sku/bin/sku.js`;
+const skuBin = path.resolve(__dirname, '../packages/sku/bin/sku.js');
 
 const runSkuScriptInDir = async (script, cwd, args = []) => {
   // When starting a dev server, return a hook to the running process
   if (/^(start|storybook|serve)/.test(script)) {
-    return gracefulSpawn(skuBin, [script, ...args], { stdio: 'inherit', cwd });
+    return gracefulSpawn(skuBin, [script, ...args], {
+      stdio: 'inherit',
+      cwd,
+      env: {
+        ...process.env,
+        STORYBOOK_DISABLE_TELEMETRY: 1,
+      },
+    });
   }
 
   // Otherwise, resolve the promise when the script finishes
   try {
-    return await exec(`${skuBin} ${script} ${args.join(' ')}`, {
+    const promise = exec(`${skuBin} ${script} ${args.join(' ')}`, {
       stdio: 'inherit',
       cwd,
       env: process.env,
-      // Elevates the buffer limit assigned to the child process.
-      // This limit is set to `200 * 1024` in node v10 and being
-      // elevated to `1024 * 1024` in node v12.
-      //
-      // See node v10: https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_child_process_exec_command_options_callback
-      // See node v12: https://nodejs.org/docs/latest-v12.x/api/child_process.html#child_process_child_process_exec_command_options_callback
-      maxBuffer: 1024 * 1024,
+      // Increased from 1024 * 1024 because Storybook can produce very large outputs.
+      // https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_process_child_process_exec_command_options_callback
+      maxBuffer: 5 * 1024 * 1024,
     });
+    await promise;
+    return { child: promise.child };
```

Comment:
This one is a little weird. Returning the promise we only get access to `stderr` and `stdout` 🤷 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/child_process.d.ts#L961-L967

Reply:
Some tests rely on the exit code 🤔 https://github.com/seek-oss/sku/actions/runs/5386488039/jobs/9776605749#step:13:952

Reply:
Yep, missed a few